src/resources/timezones.js

```diff
@@ -0,0 +1,5 @@
+import createResourceDescription from './util/createResourceDescription'
+
+export const timezones = createResourceDescription('timezones', {
+  includes: [],
```

Comment:
You can remove the includes if it's empty

Reply:
Didn't know that, thought it was a thing because I saw it also in postalCodes

Reply:
Yes, those can probably be fixed as well I think. It doesn't matter; the call will fail on the API side, but it would be cool that the SDK is the also some kind of documentation on what is possible on the API